src/primitives/transaction.h

```diff
@@ -61,13 +61,39 @@ class CTxIn
     CScript scriptSig;
     uint32_t nSequence;
 
+    /* Setting nSequence to this value for every input in a transaction
+     * disables nLockTime. */
+    static const uint32_t SEQUENCE_FINAL = 0xffffffff;
```

Comment:
Agreed. I think it's pretty sure this code or #7187 is going supersede #6312 so I'm fine with the comment being updated here. 

maybe

    /* Setting nSequence to this value for every input in a transaction
     * disables nLockTime when transaction nVersion is >=2 . */